services/cognito/app/resources/pool_resource.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class PoolResource < Cognito::ApplicationResource
-  attributes :name, :properties
+  attributes :name, :properties, :users_count
```

Comment:
Rather than this, since its not a creatable/updatable field, we should just add it the fetchable fields

```
def fetchable_fields
    super + %i[user_count]
  end
```

Reply:
Removing user_count from attributes list and just adding them to fetchable_fields doesnt seem to work. it expects attributes to be listed. 
But I have removed user_count from creatable_field and updatable_field, since its a read-only attribute.